airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1240,7 +1240,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/DestinationDefinitionIdRequestBody"
+              $ref: "#/components/schemas/OAuthDestinationConsentRequestBody"
```

Comment:
I think this approach might not work because redirect URIs have to point to `GET` request endpoints and they have to be static and defined in advance, which means we would couldn't put workspace ID, which means the redirect can't happen directly to the server, the web server needs to "inject" the workspace ID based on knowledge of the frontend session. Looped in Artem to discuss on Slack.